pandas/tests/arithmetic/test_object.py

```diff
@@ -180,3 +180,36 @@ def test_series_with_dtype_radd_timedelta(self, dtype):
 
         result = ser + pd.Timedelta('3 days')
         tm.assert_series_equal(result, expected)
+
```

Comment:
would move these to test_datetime64 (maybe rename that to test_datetime)

Reply:
These are mixed-type; it's pretty specifically the object-dtyped Index/Series/Column that these tests are addressing.